mne/time_frequency/psd.py

```diff
@@ -156,44 +162,68 @@ def compute_epochs_psd(epochs, picks=None, fmin=0, fmax=np.inf, tmin=None,
         The frequencies.
     """
     from scipy.signal import welch
-    n_fft = int(n_fft)
-    Fs = epochs.info['sfreq']
-    if picks is None:
-        picks = pick_types(epochs.info, meg=True, eeg=True, ref_meg=False,
-                           exclude='bads')
-    n_fft, n_overlap = _check_nfft(len(epochs.times), n_fft, n_overlap)
-
-    if tmin is not None or tmax is not None:
-        time_mask = _time_mask(epochs.times, tmin, tmax)
-    else:
-        time_mask = slice(None)
-
-    data = epochs.get_data()[:, picks][..., time_mask]
-    if proj:
-        proj, _ = make_projector_info(epochs.info)
-        if picks is not None:
-            data = np.dot(proj[picks][:, picks], data)
+    from .multitaper import multitaper_psd
+    from ..epochs import _BaseEpochs
```

Comment:
why is this nested? are you getting circular imports? If so, can you add that in the comments.